src/Magnum/Platform/GlfwApplication.cpp

```diff
@@ -79,7 +79,10 @@ bool GlfwApplication::tryCreateContext(const Configuration& configuration) {
         const Configuration::WindowFlags& flags = configuration.windowFlags();
         glfwWindowHint(GLFW_RESIZABLE, flags >= Configuration::WindowFlag::Resizeable);
         glfwWindowHint(GLFW_VISIBLE, !(flags >= Configuration::WindowFlag::Hidden));
-        glfwWindowHint(GLFW_MAXIMIZED, flags >= Configuration::WindowFlag::Maximized);
+        if(flags >= Configuration::WindowFlag::Maximized) {
+            /* FIXME: Workaround for https://github.com/glfw/glfw/issues/738 */
+            glfwWindowHint(GLFW_MAXIMIZED, GLFW_TRUE);
+        }
```

Comment:
@mosra I will remove the workaround. The bug has been fixed and GLFW_MAXIMIZED was added for 3.2, so the bug will not exist in any release of GLFW.